test/browser/browser.sh

```diff
@@ -4,7 +4,9 @@ set -eux
 cd "${0%/*}/../.."
 
 # HACK: ensure that critical components are up to date: https://github.com/psss/tmt/issues/682
-dnf update -y podman crun conmon criu
+if [ ! -e /sysroot/ostree ]; then
+    dnf update -y podman crun conmon criu
+fi
```

Comment:
Wondering if `bootcl status` isn't a better fit if bootc ever moves out of ostree